Platformgame/Assets/0. KimJiMin/MissileCtrl.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+// written by Lim Yeonsang
+public class MissileCtrl : MonoBehaviour
+{
+    public float _bombSpeed; //미사일 속도
+    public float MissileReferenceTime; //미사일 생성 주기
+    public GameObject Missile; //미사일 프리팹
+    
+    private readonly Vector3 _direction = Vector3.down;
+
+    //미사일움직임
+    private void FixedUpdate()
+    {
+        transform.Translate(_direction * (_bombSpeed * Time.deltaTime));
+    }
+
+    //미사일과 닿은 플레이어 사망
+    void OnTriggerEnter2D(Collider2D col)
+    {
+        if (col.gameObject.tag.Equals("Player"))
+            Destroy(col.gameObject);
```

Comment:
```suggestion
        if (col.gameObject.tag.Equals("Player"))
        {
            Destroy(col.gameObject);
        }
```
위 변수가 코드 컨벤션을 지키지 않고 작성 됨.